packages/sonarwhal/src/lib/connectors/utils/requester.ts

```diff
@@ -45,14 +73,75 @@ export class Requester {
     /** Maximum number of redirects */
     private _maxRedirects: number = 10;
 
+    /** Tries to decompress the given `Buffer` `content` using the given `decompressor` */
+    private async tryToDecompress(decompressor, content): Promise<Buffer> {
+        try {
+            const result = await decompressor(content);
+
+            return result;
+        } catch (e) {
+            return null;
+        }
+    }
+
+    /** Returns the functions to try to use in order for a given algorithm. */
+    private decompressors(algorithm: string): Array<Function> {
+        const priorities = {
+            br: 0,
+            gzip: 1,
+            deflate: 2, // eslint-disable-line sort-keys
+            identity: 3
+        };
+
+        const functions = [
+            decompressBrotli,
+            decompressGzip,
+            inflate,
+            identity
+        ];
+
+        // In case we an algorithm we don't understand
+        const priority = typeof priorities[algorithm] === 'undefined' ?
```

Comment:
From https://tools.ietf.org/html/rfc7231#section-3.1.2.2

> The "Content-Encoding" header field indicates what content codings
   have been applied to the representation, beyond those inherent in the
   media type, and thus what decoding mechanisms have to be applied in
   order to obtain data in the media type referenced by the Content-Type
   header field.  Content-Encoding is primarily used to allow a
   representation.

So, the `Content-Encoding` header can specify multiple encodings. 

E.g.: If for some reason the resource was compressed with gzip, and then, the result of that was again compressed with Brotli, you will have `Content-Encoding: gzip, br`.